redfish-core/lib/account_service.hpp

```diff
@@ -1986,15 +1986,22 @@ inline void requestAccountServiceRoutes(App& app)
             // their ConfigureSelf privilege does not apply.  In either
             // case, perform the authority check again without the user's
             // ConfigureSelf privilege.
-            if ((username != req.session->username))
+            Privileges effectiveUserPrivileges =
+                redfish::getUserPrivileges(req.userRole);
+            Privileges configureUsers = {"ConfigureUsers"};
+            bool userHasConfigureUsers =
+                effectiveUserPrivileges.isSupersetOf(configureUsers);
+            if (!userHasConfigureUsers)
             {
-                Privileges requiredPermissionsToChangeNonSelf = {
-                    {"ConfigureUsers"}};
-                Privileges effectiveUserPrivileges =
-                    redfish::getUserPrivileges(req.userRole);
 
-                if (!effectiveUserPrivileges.isSupersetOf(
-                        requiredPermissionsToChangeNonSelf))
+                // Can't modify other users
+                if (username != req.session->username)
+                {
+                    messages::insufficientPrivilege(asyncResp->res);
+                    return;
+                }
+                // Can't modify properties other than password
+                if (newUserName || enabled || roleId || locked)
```

Comment:
It's not critical, but we should add "oem" to this list.

Reply:
We don't want the operator to post the acf file?

Reply:
Done. 
https://github.com/ibm-openbmc/bmcweb/pull/159/commits/3d2788efe923706b185fe9d12447001e81e756db

Reply:
Correct.  Authorizing the "service" user is an admin function.  Also, we don't want Opeator users to be able to delete the ACF file.
The change looks good.